### PR TITLE
Add perimeter search on MediaRepository::filterSearch

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/378
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/258

### CODE REVIEW

File: Media/Repository/MediaRepositoryInterface.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function findForPaginate(PaginateFinderConfiguration $configuration, $sit
      * @param string $siteId
      * @return int
      */
-    public function count($siteId);
+    public function count($siteId, $type = null, $foldersId = null);
```
Comment (alavieille) [inline Media/Repository/MediaRepositoryInterface.php]: php doc

File: MediaModelBundle/Repository/MediaRepository.php [hunk 2]
```diff
@@ -107,13 +107,16 @@ public function findForPaginate(PaginateFinderConfiguration $configuration, $sit
      *
      * @return int
      */
-    public function count($siteId, $type = null)
+    public function count($siteId, $type = null, $foldersId = null)
```
Comment (alavieille) [inline MediaModelBundle/Repository/MediaRepository.php]: php doc

File: MediaModelBundle/Repository/MediaRepository.php [hunk 3]
```diff
@@ -172,4 +178,19 @@ protected function generateFilterKeywordLabel($label)
     {
         return array('label' => new \MongoRegex('/.*' . $label . '.*/i'));
     }
+
+    /**
+     * @param array $perimeterFolderIds
+     * @return \MongoId[][][]
```
Comment (alavieille) [inline MediaModelBundle/Repository/MediaRepository.php]: isn't a valid php doc

**Comment by ngilain** [comment]
@alavieille : update done

**Pull request merged**